README.md

```diff
@@ -1,15 +1,72 @@
 # Submariner Operator
 
-The submariner operator installs the submariner components on a kubernetes cluster.
+**Project status**: The project is meant only to be used as a development preview. Installing the operator on an Openshift cluster may disable some of the operator features. 
```

Comment:
Hmm, this is only relevant to the lighthouse/discovery part.

The other bits wont actually break anything. I prefer not to give an overall negative message. Please let's move this warning to the discovery part only.

Reply:
Moved to a separate action.